src/logic/embedding.lean

```diff
@@ -33,6 +33,13 @@ end function
 protected def equiv.to_embedding {α : Sort u} {β : Sort v} (f : α ≃ β) : α ↪ β :=
 ⟨f, f.injective⟩
 
+instance equiv.coe_embedding {α : Sort u} {β : Sort v} : has_coe (equiv α β) (α ↪ β) :=
+⟨equiv.to_embedding⟩
```

Comment:
I think you probably also want to add
```suggestion
@[simp] lemma equiv.coe_eq_to_embedding {α : Sort u} {β : Sort v} (e : α ≃ β) : ↑e = e.to_embedding :=
rfl
```
since we have lots of lemmas about `to_embedding`.

Reply:
Aha, interesting, ok. Should I also go and remove some of the below which explicitly coerce via `.to_embedding` and let the coercion do it automatically?